src/context/src/lib.rs

```diff
@@ -52,10 +53,10 @@ config!(
 /// An `Arc<Mutex<Context>>` is passed to every `Processor` run by the engine and a `&mut Context` is passed to every `State`
 /// method.
 pub struct Context {
-    // pub video_context: VideoContext,
     pub renderer: Renderer,
     pub asset_manager: AssetManager,
     pub broadcaster: Broadcaster,
+    pub audio_sink: Option<rodio::Sink>,
```

Comment:
Do we want to make context depend on rodio? I believe this is one of those use cases where we should really use the (unimplemented) shard system. Does "all" audio libs use background threads for making sound? If that is the case, maybe we could make this generic by providing a wrapper for rodio where `Sink` is `AudioSink` (which has the functions needed)? Also, my limited understanding of rodio gives me that each sink can only play one sound, and there is no way to modify the sound output other than volume.